packages/node-utilities/vitest.config.ts

```diff
@@ -0,0 +1,7 @@
+import { createVitestConfig } from '../../scripts/vitest.config';
+
+export default createVitestConfig({
+	esbuild: {
+		target: 'es2019'
+	}
+});
```

Comment:
```suggestion
export default createVitestConfig();
```

Reply:
Are you reviewing old files? because this and package.json version was updated in a previous commit

Reply:
yeah blame GH :(